dev-manuals/languages/java/code-format.md

```diff
@@ -0,0 +1,49 @@
+# Code Format
+We use [Prettier](https://prettier.io/) (more exactly the [maven-plugin](https://github.com/HubSpot/prettier-maven-plugin)) for formatting Java Code.
+Please include them as a dev dependency in frontend projects by adding this in the pom.xml:
+```xml
+<properties>
+  ...
+  <!-- By default just re-write code with prettier -->
+  <plugin.prettier.goal>write</plugin.prettier.goal>
+  ...
+</properties>
+
+<build>
+  <plugins>
+    ...
+    <plugin>
+      <groupId>com.hubspot.maven.plugins</groupId>
+      <artifactId>prettier-maven-plugin</artifactId>
+      <version>0.16</version>
+      <configuration>
+        <prettierJavaVersion>1.5.0</prettierJavaVersion>
+        <printWidth>90</printWidth>
+        <tabWidth>2</tabWidth>
+        <useTabs>false</useTabs>
```

Comment:
We use spaces?

Reply:
I do not know, I did never care about this in styling, I had always made it uniform as in the standard of my IDE. So we should use tabs and delete the tabWidth? What's your opinion?